### ISSUE TITLE
Losing connection to MC

### ISSUE THREAD

**Issue by zanz8** [Bizcuit/vscode-ampscript#46]
Hey, maybe it's just me, but for the past couple of days, VSCode seems to be losing connection to the MC server. Even after saving the settings, every time I close VSC, I have to re-enter the connection parameters. Is that possible? Thank you

**Comment by Bizcuit** [comment]
@zanz8 
I can't reproduce that. Did you create a connection with "Connection Manager" (on mac its CMD + SHIFT + P, then type MCFS and select: "MCFS: Connection Manager")?

**Comment by zanz8** [comment]
Thank you for your response.

I used "Connection Manager". Anyway I noticed that no one else seemed to have the same problem, so I wonder if this issue could be related to Copilot. The problem started when I installed the extension.

Yesterday, I decided to try uninstalling and reinstalling MCFS. Now it appears to be saving the connections again.

I can't say for certain that installing Copilot was the cause, but it does seem suspicious that the two things happened at the same time.
